drvs/bdcompat.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, plugins ? [ ], lib }:
+let
+  drvToName = drv: builtins.concatStringsSep "-" (builtins.tail (lib.strings.splitString "-" (lib.lists.last (lib.strings.splitString "/" drv))));
```

Comment:
And probably find a better way to get a name from the derivation that we receive (instead of this ugly hack). I bet there is a builtin/lib function that does it, but couldn't find any.